tools/inspect/apple_macro_check.cpp

```diff
@@ -90,8 +92,6 @@ namespace boost
                   line_start = it + 1; // could be end()
               }
           }
-
-          std::string lineloc = linelink(full_path, std::to_string(line_number));
```

Comment:
The line number should be represented by a hyperlink, however the structure of the inspect tool requires the line_number to be a size_t, so I implemented the switch in inspect.cpp for this particular case, when there are four arguments in the error report.